abfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>"meaning the intermolecular interactions turn off, keeping"</p><p> </p><p>Should that be "meaning the intermolecular interactions are turned off, whilst keeping"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>